pkg/util/kubernetes/kubelet/containers.go

```diff
@@ -63,6 +63,11 @@ func (ku *KubeUtil) ListContainers() ([]*containers.Container, error) {
 // UpdateContainerMetrics updates cgroup / network performance metrics for
 // a provided list of Container objects
 func (ku *KubeUtil) UpdateContainerMetrics(ctrList []*containers.Container) error {
+	err := providers.ContainerImpl().Prefetch()
```

Comment:
we're calling this in `ListContainers` as well, shouldn't we call it only once when creating `KubeUtil` ? is it because the cgroup pkg doesn't refresh itself unless we call `Prefetch`  ? maybe not in the scope if this PR but does it make sense to implement a refreshing mechanism in cgroup and don't let the clients care about that? it's very error prone and could also be heavy without using caching in cgroup

Reply:
cc @vboulineau

Reply:
We have two public interfaces `ListContainers` and `UpdateContainerMetrics`.
If we don't get call `Prefetch` in these functions, it basically means that we'll always return the same data, making this method useless.
Note that the change in this PR is a fix following the refactor. Before the refactor we were already refreshing in these 2 methods.
In reality, this `UpdateContainerMetrics` method should not exist, as, in any case, we refresh everything.